src/services/alchemy.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react';
+import { Alchemy, Network, Nft } from 'alchemy-sdk';
+
+const alchemy = new Alchemy({
+  apiKey: process.env.EXPO_PUBLIC_ALCHEMY_API_KEY,
+  network: Network.ETH_MAINNET,
+});
+
+export function useNFTsForAddress(address: string) {
```

Comment:
[nit] inconsistent casing
```suggestion
export function useNftsForAddress(address: string) {
```